src/Workspaces/VisualBasic/Portable/Simplification/AbstractVisualBasicSimplifier.AbstractExpressionRewriter.vb

```diff
@@ -31,6 +31,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Simplification
                 End Get
             End Property
 
+            Private Shared Function GetParentNode(node As SyntaxNode) As SyntaxNode
+                Dim expression = TryCast(node, ExpressionSyntax)
+                If expression IsNot Nothing Then
+                    Return GetParentNode(expression)
+                End If
+
+                If node.IsKind(SyntaxKind.SimpleArgument) AndAlso node.IsParentKind(SyntaxKind.TupleExpression) Then
+                    Return node.Parent
+                End If
+
+                If node.IsKind(SyntaxKind.NamedFieldInitializer) Then
+                    Return node.Parent
+                End If
+
+                Return Nothing
```

Comment:
Removed this method.

---
In reply to: [113553557](https://github.com/dotnet/roslyn/pull/18930#discussion_r113553557) [](ancestors = 113553557)